packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -95,6 +95,25 @@ pub struct ExperimentConfig {
     clap(global = true, short = 'w', long, default_value_t = num_cpus::get(), validator = at_least_one, env = "HASH_WORKERS")
     )]
     pub num_workers: usize,
+
+    #[cfg_attr(feature = "clap", clap(global = true, long, default_value_t = 0))]
+    /// Initial size of the V8 heap.
```

Comment:
I guess this is in Bytes? Why choose bytes in the first place, I think MB would be way more sensible.

Reply:
I just mimicked the V8 interface but yes MB for users seems like a good idea.

Reply:
Changed to MB in https://github.com/hashintel/hash/pull/446/commits/3fa505a31887f9dd587def8c8d32b66af320317c

Reply:
You only did that for the max-constraint, not for the initial heap size.

Reply:
Completely rephrase in https://github.com/hashintel/hash/pull/446/commits/4995421adf9d244153b456545155ac245df3796b